dev-haskell/hoogle/files/hoogle-5.0.18-disable-network-tests.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/src/Action/CmdLine.hs b/src/Action/CmdLine.hs
+index d3bccb4f..948c775d 100644
+--- a/src/Action/CmdLine.hs
```

Comment:
Can you send this patch upstream? I'd like to avoid carrying patches that will probably need to be rebased.

Reply:
Yes I plan to submit a PR upstream.

Reply:
[PR sumbitted](https://github.com/ndmitchell/hoogle/pull/365) upstream.